packages/editor-ui/src/stores/cloudPlan.store.ts

```diff
@@ -0,0 +1,61 @@
+import { computed, reactive } from 'vue';
+import { defineStore } from 'pinia';
+import type { CloudPlanState } from '@/Interface';
+import { useRootStore } from '@/stores/n8nRoot.store';
+import { useSettingsStore } from '@/stores/settings.store';
+import { useUsersStore } from '@/stores/users.store';
+import { getCurrentPlan } from '@/api/cloudPlans';
+
+const DEFAULT_STATE: CloudPlanState = {
```

Comment:
tricky to have this as default state.. often it's better to have empty default state.. what happens if api call fails? if the BE is down for whatever reason? then would not this default show for non-trial users?

Reply:
if the request fails or the back-end is down, the component is not rendered, as at that point I have not updated the store. Tested by throwing an exception in the getOwnerCurrentPLan method instead of the data.

Reply:
can we still default to null still? some of the data is here still valid like executions count or the metadata or planId.. would rather avoid future bugs around this by having a clear not yet loaded state.